### ISSUE TITLE
Error connecting to "api.intercom.iohttps"

### ISSUE THREAD

**Issue by jfeldstein** [jkeyes/python-intercom#120]
Attempting to search for a list of Companies, by tag_id, I get:

```
  File "/Users/jordan/Code/kiddom/kiddom-crm/utils/web_apis/intercom_wrapper.py", line 137, in get_schools_with_tag
    return [s.company_id for s in schoolObjs]
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/six.py", line 558, in next
    return type(self).__next__(self)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/collection_proxy.py", line 54, in __next__
    self.get_next_page()
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/collection_proxy.py", line 71, in get_next_page
    return self.get_page(self.next_page)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/collection_proxy.py", line 81, in get_page
    response = self.client.get(url, params)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/client.py", line 89, in get
    return self._execute_request(req, params)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/client.py", line 82, in _execute_request
    result = request.execute(self.base_url, self._auth, params)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/request.py", line 23, in execute
    return self.send_request_to_path(base_url, auth, params)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/intercom/request.py", line 57, in send_request_to_path
    auth=auth, verify=certifi.where(), **req_params)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/requests/api.py", line 53, in request
    return session.request(method=method, url=url, **kwargs)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/requests/sessions.py", line 468, in request
    resp = self.send(prep, **send_kwargs)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/requests/sessions.py", line 576, in send
    r = adapter.send(request, **kwargs)
  File "/Users/jordan/Code/kiddom/kiddom-crm/env/lib/python2.7/site-packages/requests/adapters.py", line 437, in send
    raise ConnectionError(e, request=request)
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='api.intercom.iohttps', port=443): Max retries exceeded with url: //api.intercom.io/companies?per_page=15&page=2 (Caused by NewConnectionError('<requests.packages.urllib3.connection.VerifiedHTTPSConnection object at 0x10b9f5550>: Failed to establish a new connection: [Errno 8] nodename nor servname provided, or not known',))
```

Notice it's trying to call `host='api.intercom.iohttps'` and url `//api.intercom.io/companies...`

The https is in the wrong place.

**Comment by jfeldstein** [comment]
I've opened a PR which addresses this issue. Please take a look and let me know how I can get it merged! https://github.com/jkeyes/python-intercom/pull/121

**Comment by jkeyes** [comment]
Fixed in #125 and merged into version-3 branch.
